### PR TITLE
[Docs] Use NAV-dekorator for Page examples

### PR DESCRIPTION
**Pull request by KenAJoh**
### Description

Creates a more "realistic" scenario for use of Page-component

### Component Checklist 📝

- [ ] JSDoc
- [x] Examples
- [ ] Documentation / Decision Records
- [ ] Storybook
- [ ] Style mappings (`@navikt/core/css/config/_mappings.js`)
- [ ] Component tokens (`@navikt/core/css/tokens.json`)
- [ ] CSS class deprecations (`@navikt/aksel-stylelint/src/deprecations.ts`)
- [ ] Exports (`@navikt/core/react/src/index.ts` and `@navikt/core/react/package.json`)
- [ ] New component? CSS import (`@navikt/core/css/index.css`)
- [ ] Breaking change? Update migration guide. Consider codemod.
- [ ] Changeset (Format: `<Component>: <gitmoji?> <Text>.` E.g. "Button: :sparkles: Add feature xyz.")

### CODE REVIEW

File: aksel.nav.no/website/components/website-modules/examples/__parts/Dekorator.tsx [hunk 1]
```diff
@@ -8,13 +8,35 @@ function Footer() {
...
+            #eee,
+            #eee 10px,
+            #fff 10px,
+            #fff 20px
+          );`}
+      </style>
+      {children}
+    </div>
+  );
+}
```
Comment (HalvorHaugan) [inline aksel.nav.no/website/components/website-modules/examples/__parts/Dekorator.tsx]: I think you need to put it in a separate file (ref. ci errors)

File: aksel.nav.no/website/components/website-modules/examples/__parts/PageDemoContent.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+function Content({ children }: { children: React.ReactNode }) {
+  return (
+    <div className="decorator-content">
+      <style>
+        {`.decorator-content {
+          width: 100%;
+          padding-block: 5rem;
+          padding-inline: 0.25rem;
+          height: 100%;
```
Comment (HalvorHaugan) [inline aksel.nav.no/website/components/website-modules/examples/__parts/PageDemoContent.tsx]: I don't think width and height is needed
```suggestion
          padding-block: 5rem;
          padding-inline: 0.25rem;
```

File: aksel.nav.no/website/pages/eksempler/primitive-page/background.tsx [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-        padding="8"
-        paddingBlock="16"
-      >
-        <Page.Block gutters>Content</Page.Block>
-      </Box>
+    <Page background="bg-subtle" footer={<Footer />}>
+      <Header />
+      <Page.Block as="main" width="xl" gutters>
+        <Content>
+          <Heading level="1" size="large" spacing>
```
Comment (HalvorHaugan) [inline aksel.nav.no/website/pages/eksempler/primitive-page/background.tsx]: Consider removing `spacing` on Heading on all the examples

File: aksel.nav.no/website/pages/eksempler/primitive-page/demo.tsx [hunk 4]
```diff
@@ -1,32 +1,28 @@
-import { Box, Page } from "@navikt/ds-react";
+import { Heading, Page } from "@navikt/ds-react";
+import { Content } from "@/web/examples/__parts/PageDemoContent";
```
Comment (HalvorHaugan) [inline aksel.nav.no/website/pages/eksempler/primitive-page/demo.tsx]: Alias in imports from __parts doesn't work
```suggestion
import { Content } from "../../../components/website-modules/examples/__parts/PageDemoContent";
```

File: aksel.nav.no/website/pages/eksempler/primitive-page/demo.tsx
Comment (HalvorHaugan) [inline aksel.nav.no/website/pages/eksempler/primitive-page/demo.tsx:1]: Should we write in the desc that you can use Page.Block for header and footer if you're not using the decorator, or do you think people will understand that?

Comment (KenAJoh) [reply aksel.nav.no/website/pages/eksempler/primitive-page/demo.tsx:1]: Can add some text
```
"`Page.Block` kan også brukes i header og footer for å sikre at innholdet er sentrert og har samme maksbredde som resten av innholdet."
```

**Review by HalvorHaugan — DISMISSED** [review]

File: aksel.nav.no/website/components/website-modules/examples/__parts/PageDemoContent.tsx [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
```
Comment (HalvorHaugan) [inline aksel.nav.no/website/components/website-modules/examples/__parts/PageDemoContent.tsx]: Looks like you need to remove this
```suggestion
```

**Review by HalvorHaugan — APPROVED** [review]

**Pull request merged**